cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactFile.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.internal.app.runtime.artifact;
+
+import co.cask.cdap.api.artifact.ArtifactId;
+import co.cask.cdap.proto.Id;
+
+/**
+ * Helper methods related to handling of {@link Id.Artifact}s.
+ */
+public class ArtifactFile {
```

Comment:
final class and private constructor for util class. Also, better not to have such specific util class. Maybe you can have a `Artifacts` util class which has a collection of helper methods for Artifact related operations.